eng/dockerfile-templates/aspnet/Dockerfile

```diff
@@ -15,15 +15,24 @@ RUN dism /Online /Quiet /Enable-Feature /All /FeatureName:IIS-WebServerRole {{if
 {{if PRODUCT_VERSION != "3.5"
 :
 # Install Roslyn compilers
```

Comment:
`# Install 2.9.0 Roslyn compilers`

Reply:
Updated the comments: # Install 2.9.0 Roslyn compilers